### ISSUE TITLE
Extend support for Python 3.9

### ISSUE THREAD

**Issue by santisoler** [fatiando/maintenance#5]
**Description**

Extend support of all our libraries to Python 3.9. This involves:

1. Adding CI jobs with Python 3.9 so we know things work. This usually only involves adding "3.9" to the list of Python versions in the configuration files.
2. Add the 3.9 tag to `setup.py`

**Apply changes to**

Libraries:

- [ ] [boule](https://github.com/fatiando/boule)
- [ ] [erizo](https://github.com/fatiando/erizo)
- [ ] [harmonica](https://github.com/fatiando/harmonica)
- [ ] [pooch](https://github.com/fatiando/pooch)
- [ ] [rockhound](https://github.com/fatiando/rockhound)
- [ ] [verde](https://github.com/fatiando/verde)

Other repositories:

- [ ] [.github](https://github.com/fatiando/.github)
- [ ] [contributing](https://github.com/fatiando/contributing)
- [ ] [continuous-integration](https://github.com/fatiando/continuous-integration)

**Further instructions**

In order to solve this issue we need to start opening Pull Requests on each
repository listed above. Optionally, we can open Issues on each repository if
further discussion specific to that repository is needed.

Remember to mention this Issue on every Issue or Pull Request opened on each
repository for keeping a record by adding something like:

```
Related to fatiando/maintenance#XX
```

A quick checklist:

1. Open an Issue on each repository listed above (optional). Mention this Issue
   on them.
2. Open a Pull Request on the same repository to tackle the Issue. Mention this
   Issue on them.
3. Check-off the repository on the list above once the Pull Request is merged.
4. Close this issue when all items are checked-off.

**We want your help**

We know that maintenance tasks are very demanding, so we don't expect a single
person to tackle this issue by themselves. Any help is very welcomed, so please
comment below that you want to take care of the changes on any repository and
we will assign it to you.

**Comment by santisoler** [comment]
Duplicate of #14
